src/ProjectTemplates/Web.ProjectTemplates/content/SignalR-CSharp/Hubs/StreamingTimerHub.cs

```diff
@@ -0,0 +1,38 @@
+using System;
+using System.Threading;
+using System.Threading.Channels;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.SignalR;
+
+namespace Company.WebApplication1
+{
+    public class StreamingTimerHub : Hub
+    {
+        public ChannelReader<string> ServerTimer(CancellationToken token)
```

Comment:
I recommend the parameter name `cancellationToken` (here, and everywhere). The short name `token` is ambiguous.

Reply:
Don't use ChannelReader, use IAsyncEnumerable.